ccip/config/evm/Monad_Testnet.toml

```diff
@@ -0,0 +1,23 @@
+ChainID = '10143'
+# finality_depth was: 0
+FinalityDepth = 10
+# block_time was:  1s, adding 1 second buffer  
+LogPollInterval = '2s'
+# No WS Support
+LogBroadcasterEnabled = false
+FinalityTagEnabled = false
+# finality_depth * block_time / 60 secs = < 1 min (finality time)
+NoNewFinalizedHeadsThreshold = '1m'
```

Comment:
nit: Should we also set NoNewHeads to a lower value?

Reply:
Good catch. added!